scripts/seed-database/upsert-production-flows.js

```diff
@@ -72,7 +71,12 @@ const LOCAL_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
 
     await localClient.request(
       insertFlowsMutation,
-      { flows: productionFlows || [], }
+      {
+        flows: productionFlows?.map(flow => ({
+          ...flow,
+          version: 1,
```

Comment:
pizza seeds seem to be working fine here, but i'm still a little confused about this line: if you're always setting `version: 1` here, won't that fail to then link up with the most recent operation that you've fetched & inserted? or has that operation's version also been overwritten to 1?

Reply:
This script isn't doing an UPSERT anymore but instead a DELETE-THEN-INSERT.